beacon-chain/state/stateutil/field_root_validator_test.go

```diff
@@ -58,3 +58,20 @@ func TestHashValidatorHelper(t *testing.T) {
 		require.Equal(t, [32]byte{}, roots[i])
 	}
 }
+
+func BenchmarkTestValidatorRegistryRoot(b *testing.B) {
+	valList := make([]*ethpb.Validator, 1_000_000)
+	for i := range valList {
+		valList[i] = &ethpb.Validator{
+			PublicKey:        []byte{byte(i)},
+			EffectiveBalance: uint64(i),
+			PrincipalBalance: uint64(i),
+		}
+	}
+	b.Run("100k validators", func(b *testing.B) {
```

Comment:
```suggestion
	b.Run("1 mil validators", func(b *testing.B) {
```

Seems 1 million is right